src/components/faqbox/index.js

```diff
@@ -1,26 +1,55 @@
-import React from 'react';
-import styled from 'styled-components'
-;
+import React from "react";
+import styled from "styled-components";
 const Box = styled.div`
-    background-color: white;
-    box-shadow: 2px 10px 30px rgba(0, 0, 0, 0.25);
-    border-radius: 7px;
-    text-align: left;
-    width: 20rem;
-    margin: 5rem;
-    padding: 2rem;
+  background-color: white;
+  box-shadow: 2px 10px 30px rgba(0, 0, 0, 0.25);
+  border-radius: 7px;
+  text-align: left;
+  width: 20rem;
+  margin: 5rem;
+  padding: 2rem;
+  font: Roboto;
+  font-style: normal;
```

Comment:
Can remove

Reply:
Same here, should only apply to the font-style: normal;

Reply:
remove font and font-style? or margin and padding as well?